### DISCUSSION TITLE
Slider End count shouldn't take up 2 columns in the score table

### DISCUSSION THREAD

**Discussion by 424ever** [ppy/osu#29904] [Ideas] [answered]
When looking at the scores on [this map](https://osu.ppy.sh/beatmapsets/847776#osu/1772923) in game, there are 2 columns labeled as "Slider End". I think these should be combined into a single column.
![image](https://github.com/user-attachments/assets/10dfe2b2-fa10-4c85-b2ec-a9a00e22611e)
There is (currently) only one score which uses the left column, which was set on Lazer with the Classic mod.
![image](https://github.com/user-attachments/assets/a9f73333-714a-4c9b-affa-e4bf14329b0e)

**Comment by peppy** [comment, marked as answer]
Thanks for the report. Tracking at https://github.com/ppy/osu/issues/29911.
